blog-pipeline/tests/test_graphics_source.py

```
@@ -0,0 +1,91 @@
+"""Tests for source attribution removal from generated graphics (#119)."""
+
+import sys
+from pathlib import Path
+
+# Ensure the skills directory is importable
+_REPO_ROOT = Path(__file__).resolve().parent.parent
+sys.path.insert(0, str(_REPO_ROOT / "skills" / "blog-workflow"))
+
+from graphics_generator import _strip_source_attribution
```

Ruff (0.15.7)

[error] Module level import not at top of file

(E402)